src/MsixPackageProject/Package.appxmanifest

```diff
@@ -9,13 +9,13 @@
   IgnorableNamespaces="uap rescap">
 
   <Identity
-    Name="Microsoft.AzureRTOSTraceX"
-    Publisher="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US"
+    Name="Eclipse.EclipseThreadXTraceX"
+    Publisher="CN=Eclipse Foundation, O=Eclipse Foundation, L=Ottawa, S=Ontario, C=CA"
```

Comment:
I believe changing the publisher identity in the package.appxmanifest will prevent the store package (.msix) from being installable. It needs to match the certificate.

Reply:
reverted the publisher